sqlstore/src/main/resources/db/migration/V0030__project_shortName.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE Project ADD COLUMN shortName varchar(20) DEFAULT NULL;
```

Comment:
Should be varchar(5). Can we default current project shortNames to their alias truncated to 5 characters too?